pkg/agent/server.go

```diff
@@ -97,6 +97,11 @@ func (s *Server) ServeDebug(w http.ResponseWriter, req *http.Request) {
 		Stderr: false,
 		TTY:    true,
 	}
+	LxcfsEnabled = req.FormValue("lxcfsEnabled")
+	if len(LxcfsEnabled) < 1 {
+		http.Error(w, "LxcfsEnabled must be provided", 400)
```

Comment:
It is better to be default to `false`, because the client could be an  old version of `kubectl-debug`

Reply:
that's true, i will fix it soon.